template.py

```diff
@@ -78,7 +80,19 @@ def render_file(args, env, state, file):
 
         # process template file
         input = in_stream.read()
-        template = env.from_string(input)
+        template = env.from_string(input())
```

Comment:
`input` is not a function, so should stay as what it was. Are you getting a type error for this at runtime?

```suggestion
        template = env.from_string(input)
```